dcs-packaging-tool/dcs-packaging-tool-model/src/main/java/org/dataconservancy/packaging/tool/model/dprofile/PropertyConstraint.java

```diff
@@ -0,0 +1,22 @@
+package org.dataconservancy.packaging.tool.model.dprofile;
+
+/**
+ * A PropertyConstraint represents restrictions on the cardinality and type of a
+ * property in a node.
+ */
+public interface PropertyConstraint {
```

Comment:
Maybe convenience methods `isOptional()` where `minCardinality` is 0 and `maxCardinality` is (what is used to represent unconstrained) `Integer.MAX_VALUE`?

And perhaps `isRequired()` where `minCardinality` is greater than 0?